ansible_collections/arista/avd/docs/getting-started/avd-aap.md

```diff
@@ -0,0 +1,400 @@
+
+=== "Inventories - Add"
+
+    In the new pane, click on `Add` and select `Add inventory`.
+
+    ![Add inventory](../_media/getting-started/aap-avd/select-add-inv.png)
+
+=== "Inventories - Save"
+
+    Give the inventory an appropriate `Name` and click `Save`.
+
+    ![Create inventory](../_media/getting-started/aap-avd/save-inv.png)
+
+### Inventory Sources
+
+At this point, we have an inventory with no hosts. This is where inventory sources come into play. Like most things, we have a series of options. We can leverage an inventory source from Cloud providers, virtualization platforms, or, in our case, directly from our project.
+
+=== "Click on Sources"
+
+    On the center pane, click on `Sources`.
+
+    ![Select sources](../_media/getting-started/aap-avd/select-sources.png)
+
+=== "Sources - Add"
+
+    - In the new pane, click on `Add`.
+    - A new pane is shown but similar to before, fill in an appropriate `Name` and select `Source from a Project` under `source`.
+    - Under `Source details` click on the search icon under `Project`
+      - Select your newly created project
+    - Under `Source details`, if the `Inventory file` drop-down does not show your inventory, feel free to enter it manually.
+    - Under `Update options`:
+      - `Overwrite` is checked. Overwrite will ensure our inventory source manages additions and removal of hosts and groups.
+      - `Update on launch` is checked.
+    - Scroll down and click `Save`.
+
+    ![Create source](../_media/getting-started/aap-avd/create-source.png)
+
+=== "Sources - Sync"
+
+    In the new pane, click on `Sync` to update your inventory.
+
+    ![Inventory sync](../_media/getting-started/aap-avd/inv-sync.png)
+
+### View the Inventory
+
+There are a few locations to view the inventory, but for simplicity, we can view it by clicking on `Hosts` on the left pane.
+
+=== "Click on Sync Job"
+
+    - On the left pane, click on `Jobs`.
+      - You should see an inventory sync running or complete.
+    - Click on the inventory sync job.
+
+    ![Inventory job sync](../_media/getting-started/aap-avd/inv-job-sync.png)
+
+=== "Sync job output"
+
+    Towards the bottom of the output, we see that AAP successfully parsed our inventory. In this case, we have loaded nine groups and seven hosts.
+
+    ![Sync job output](../_media/getting-started/aap-avd/inv-job-output.png)
+
+=== "Hosts"
+
+    There are a few locations to view the inventory, but for simplicity, we can take a look by clicking on `Hosts` on the left pane.
+
+    ![View inventory](../_media/getting-started/aap-avd/all-hosts.png)
+
+## Job templates and workflow templates
+
+One thing that may need some clarification is the naming of "job templates." These map to playbooks within our project. There is also an option to build workflow templates: a series of job templates with some control logic built-in. For this example, we will use a job template to build and deploy our node configurations with CVP.
```

Comment:
"job templates".

Reply:
Needs to go within the quote :) The final period or comma goes inside the quotation marks, even if it is not a part of the quoted material, unless the quotation is followed by a citation.  If a citation in parentheses follows the quotation, the period follows the citation.